ArduPlane/Parameters.cpp

```diff
@@ -1543,6 +1543,17 @@ void Plane::load_parameters(void)
 #if AP_FENCE_ENABLED
     AP_Param::convert_class(g.k_param_fence, &fence, fence.var_info, 0, 0, true);
 #endif
+  
+    // PARAMETER_CONVERSION - Added: Dec 2023
+    // Convert _CM (centimeter) parameters to meters and _CD (centidegrees) parameters to meters
+    {
+        const AP_Param::ConversionInfo centimeters_conversion_info[] {
+            { Parameters::k_param_pitch_trim_cd_old, 0, AP_PARAM_FLOAT, "PITCH_TRIM" },
```

Comment:
I think the type here is the old param type.
```suggestion
            { Parameters::k_param_pitch_trim_cd_old, 0, AP_PARAM_INT16, "PITCH_TRIM" },
```

Reply:
Done